.dialyzer_ignore.exs

```diff
@@ -0,0 +1,13 @@
+[
+  {":0:unknown_function Function Common.Validation.Atom.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.BitString.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.Float.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.Function.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.Integer.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.List.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.Map.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.PID.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.Port.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.Reference.__impl__/1 does not exist."},
+  {":0:unknown_function Function Common.Validation.Tuple.__impl__/1 does not exist."}
+]
```

Comment:
not a huge fan of these ignores. imo we should only be ignoring an issue if its caused by a dep. if its our fault, we should fix it instead

Reply:
yeah, they are quite ugly.
But I don't see the issue that the missing implementations could cause and I favor the protocols for validation.